store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreTest.java

```diff
@@ -256,11 +256,6 @@ public void shouldNotAllowOverwritingOfGraphWithinFederatedScope() throws Except
         }
     }
 
```

Comment:
In order to replace the outgoing test, it would be good to add something like:
```suggestion
    @Test
    public void shouldThrowAppropriateExceptionWhenHandlingAnUnsupportedOperation() {
        // Given
        Operation op = new OperationImpl();
        // When
        try {
            store.handleOperation(op, new Context());
            fail("Exception expected");
        } catch (final OperationException e) {
            fail("Expected an UnsupportedOperationException rather than an OperationException");
        } catch (final UnsupportedOperationException e) {
            // Then
            assertEquals("Operation class uk.gov.gchq.gaffer.operation.impl.OperationImpl is not supported by the FederatedStore.", e.getMessage());
        }
    }
```